install/kots/manifests/kots-preflight.yaml

```diff
@@ -7,6 +7,19 @@ metadata:
   name: gitpod
 spec:
   collectors:
+    - run:
+        collectorName: database
+        image: eu.gcr.io/gitpod-core-dev/build/kots-config-check/database:sje-kots-config-check.9
```

Comment:
Will this work in an air-gap setting as well? I guess that KOTS will handle this. However, we need to make sure it will before we can merge this.

Reply:
Yes it will. It's exactly the same as the [Installer job](https://github.com/gitpod-io/gitpod/blob/main/install/kots/manifests/gitpod-installer-job.yaml#L31)